sai_p4/instantiations/google/test_tools/set_up_bmv2.cc

```diff
@@ -10,7 +10,7 @@
 
 namespace sai {
 
-using ::orion::p4::test::Bmv2;
```

Comment:
Same as the other one, this will not compile

Reply:
> Same as the other one, this will not compile

This file is also not defined in BUILD.bazel

Reply:
Same as the other one, can you check if its in pins_infra, if yes, why it got missed in sonic-pins.

Reply:
> Same as the other one, can you check if its in pins_infra, if yes, why it got missed in sonic-pins.

This file is not added in the BUILD file in Pins_infra.